runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -4297,6 +4297,7 @@ void JitShutdown(J9JITConfig * jitConfig)
 
    J9JavaVM * javaVM = jitConfig->javaVM;
    J9VMThread *vmThread = javaVM->internalVMFunctions->currentVMThread(javaVM);
+   TR::CompilationInfo *compInfo = TR::CompilationInfo::get(jitConfig);
```

Comment:
This variable appears unused if `defined(J9VM_OPT_JITSERVER)` which may yield a warning/error.